product_state/views/product_views.xml

```diff
@@ -21,7 +21,7 @@
         <field name="arch" type="xml">
     	    <field name="type" position="after">
                 <field name="state"/>
-	    </field>
+            </field>
```

Comment:
Why this extra space?

Reply:
Good catch, thanks :)
Actually it is not an extra space: I had converted a tab to 4 spaces and it seems github does not show them the same way. Your comment made me realize the opening ``<field>`` tag was not correctly indented => I have just pushed 0d332ec making sure the whole file has no tab.